tensorrt_llm/_torch/pyexecutor/request_utils.py

```
@@ -70,6 +75,46 @@ def attach_py_objects_to_requests(requests: List, py_request_objects: Tuple) ->
                     setattr(item.request, attr_name, py_obj)
 
 
+def build_no_fitting_reqs_diagnostic(active_requests, kv_cache_manager) -> str:
+    """Diagnostic for the no-fitting-reqs case: active state distribution
+    plus KV pool / IndexMapper occupancy."""
+    state_counts: Dict[str, int] = {}
+    for req in active_requests:
+        try:
+            name = LlmRequestState(req.state_value).name
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)